theta/include/theta_a_not_b_impl.hpp

```diff
@@ -79,28 +77,22 @@ compact_theta_sketch_alloc<A> theta_a_not_b_alloc<A>::compute(const theta_sketch
     // scan A lookup B
     for (auto key: a) {
       if (key < theta) {
-        if (!update_theta_sketch_alloc<A>::hash_search(key, b_hash_table, lg_size)) keys[count++] = key;
+        if (!update_theta_sketch_alloc<A>::hash_search(key, b_hash_table.data(), lg_size)) keys[count++] = key;
```

Comment:
Is hash_search() ever used on a raw pointer after this refactor? If no, we should just update its signature to take a vector.